tools/experiments/experiments-config.js

```diff
@@ -185,12 +161,6 @@ export const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/23568',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/24165',
   },
-  {
-    id: 'fix-inconsistent-responsive-height-selection',
-    name: 'Fix inconsistent responsive height selection.',
-    spec: 'https://github.com/ampproject/amphtml/issues/24166',
-    cleanupIssue: 'https://github.com/ampproject/amphtml/issues/24167',
-  },
```

Comment:
`cleanupIssue` #24167

Reply:
This one doesn't have a similar code removal?

Reply:
No, it was only passing its value to a backend.